build.yml

```diff
@@ -0,0 +1,68 @@
+parameters:
+  name: ''
+  queue: ''
+  platform: ''
+
+phases:
+
+- phase: ${{ parameters.name }}
+  queue:
+    name: ${{ parameters.queue }}
+    parallel: 3
+    matrix:
+      Python35:
+        PYTHON_VERSION: '3.5'
+      Python36:
+        PYTHON_VERSION: '3.6'
+      ${{ if eq(parameters.platform, 'Linux') }}:
+        Python37:
+          PYTHON_VERSION: '3.7'
+      # Python38-dev:
+      #   PYTHON_VERSION: '>= 3.8.0a'
+
+  steps:
+
+    - task: UsePythonVersion@0
+      inputs:
+        versionSpec: '$(PYTHON_VERSION)'
+        architecture: 'x64'
+      displayName: Use Python version $(PYTHON_VERSION)
+    
+    - ${{ if eq(parameters.platform, 'Linux') }}:
+      - bash: |
+          sudo apt-get -yq update
+          sudo apt-get -yq install libmemcached-dev
+        displayName: Setup Linux environment
+
+    - script: |
+        python -m pip install --upgrade pip setuptools wheel
+        pip install unittest-xml-reporting
+      displayName: Setup Python environment
+
+    - script: |
+        pushd tests
+        pip install -r requirements/py3.txt
+        pip install -e ..
+        python runtests.py --noinput --settings test_sqlite_vsts_ci --verbosity 2
+        exit 0

```

Comment:
Are the `exit 0` statements required here and below? They are not present in the `Setup Python environment` or the `Setup Linux environment` sections above, and isn't a return code of 0 implicit if there is no failure?

Reply:
Good catch, this should be a `continueOnError` directive in the YAML, not a `return 0` in the script.

If the code under test breaks, the CI still needs to continue. By default we look at the last exit code to determine whether to continue, and the test runner returns non zero if any tests fail.